actions/files_upload.py

```diff
@@ -6,8 +6,8 @@ class FilesUploadAction(SlackAction):
     def run(self, **kwargs):
         # https://requests.readthedocs.io/en/master/user/quickstart/#post-a-multipart-encoded-file
         files = None
-        if 'file_path' in kwargs:
-            if 'file' in kwargs and kwargs['file']:
+        if 'file_path' in kwargs and kwargs['file_path'] != '':
+            if 'file' in kwargs and kwargs['file'] != '':
```

Comment:
Empty strings in Python test false-y, so you can just do:

```suggestion
        if 'file_path' in kwargs and kwargs['file_path']:
            if 'file' in kwargs and kwargs['file']:
```

Reply:
Learn something new every day, fixed